buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -346,8 +349,14 @@ public String toString() {
         Provider<DockerSupportService> dockerSupport
     ) {
         return project.getTasks().register(destructiveDistroTestTaskName(distribution), Test.class, t -> {
+            // Only run tests for the current architecture
             // Disable Docker distribution tests unless a Docker installation is available
-            t.onlyIf(t2 -> distribution.getType() != Type.DOCKER || dockerSupport.get().getDockerAvailability().isAvailable);
+            t.onlyIf(t2 -> {
```

Comment:
FYI, you can add multiple `onlyIf` conditions, which I think i easier to reason about than having a single more complex conditional. Basically I only run this task if "docker works" **and** "it's the right architecture".

Reply:
Nice, TIL.